### DISCUSSION TITLE
Slanted parts of a Block in MC Bedrock

### DISCUSSION THREAD

**Discussion by ZenDeo** [bridge-core/bridge.#318] [Q&A] [answered]
Has anyone tried making a block with some parts slanted/diagonal? 

I've started coding last 2 days, and just now I've encountered a problem in putting in the geo file of a block. The code is the same with other blocks I've made that have worked, but when I open this specific block in Minecraft, it shows an update block.

Is the model unsupported? Do I have to remodel the block? Or is there some code that I have to put in for it to work?

I have a tweet about it. Twitter: @ZenDeoIsHere

![image](https://user-images.githubusercontent.com/78517606/106835753-dce39600-66d2-11eb-824d-cfe5b8b34f5c.png)

![image](https://user-images.githubusercontent.com/78517606/106835811-f1c02980-66d2-11eb-9cce-af7aade060cd.png)

![image](https://user-images.githubusercontent.com/78517606/106835875-0bfa0780-66d3-11eb-937d-91559ba5cf6a.png)

**Comment by solvedDev** [comment, marked as answer]
Make sure that your rotation is on bones and not cubes. Afaik, cube rotation is currently not supported on block models.
